docs/guides/reference/ard-processing-extent.md

```diff
@@ -0,0 +1,93 @@
+* - Gulf of Carpentaria
+  - 101-069, 53LRC, 54LTH 
+* - Arafura Sea
+  - 102-067, 104-067, 106-067, 107-068, 107-069, 52LEM, 52LEN, 52LFP, 52LHP, 53LLJ, 53LPJ, 53LQH, 53LQJ 
+* - Timor Sea
+  - 51LVD, 51LWD, 51LWF, 51LXE, 51LXF, 51LYF, 51LZF, 52LBL, 52LCL, 52LDK 
+* - Indian Ocean
+  - 116-075, 49HGB, 49HGC, 49JFL, 49KGS, 49KHT, 50HKG, 50HKH, 50HKJ, 50HMF, 50HNF, 50JKL, 50KKC 
+* - Great Australian Bight
+  - 52HBI, 52HEK, 52HFK, 53HKD, 53HLC, 53HNA 
+* - Augusta, Western Australia
+  - 113-084 
+* - Tasman Sea
+  - 089-088, 089-091, 090-091, 55GFM 
+* - Bass Strait
+  - 54HXB 
+
+:::
+
+## Quality and metadata 
+
+The quality of ancillary inputs used in ARD generation in offshore locations is limited in comparison to the higher quality ancillary inputs used on continental Australia. Quality information for ancillary inputs is found in the proc-info (processing information) YAML metadata file within the dataset. The tier list is USER, FALLBACK, and DEFINITIVE, with the latter being the highest level of quality. Unfortunately, this information is not available to users via the datacube, so we have added an additional metadata tag to delineate data delivered in this update: `final_ancillaries = "nonstandard"`
+
+```python
+dc.find_datasets(product="ga_ls8c_ard_3", limit=10, final_ancillaries="nonstandard")
+```
+
+## Geometric quality assessment issues 
+
+[ARD](/guides/about/glossary/#ard) is a Surface Reflectance product which is derived from USGS or ESA level 1 products. The level 1 product is geometrically corrected such that the product is 'ortho-rectified' which results in enabling acquisitions from different dates to be spatially overlaid for assessment through time. The ARD product provides a comprehensive quality assurance of geometric correction known as GQA, which is found in the dataset and processing information metadata.
+
+This update features the addition of Landsat scenes and Sentinel 2 tiles over islands and reefs. Some of these Islands, such as Christmas Island or Islands South of Papua New Guinea, are at extreme high or low latitudes and are affected by persistent cloud. This presents challenges when correlating the reference image against source images, to provide statistical geometric comparison results.
+
+## BRDF quality issues 
+
+[BRDF](/guides/about/glossary/#brdf) is a solar illumination correction which adjusts reflectance intensity in differing amounts in differing directions. The light that comes from the Sun, reflects off the Earth and is received by the sensor is adjusted based on viewing/solar angle geometries of MODIS data.
+
+Sometimes over offshore locations we have found BRDF values to be unphysical. We have implemented a BRDF solution over the offshore locations, which screens out unphysical BRDF values and excludes them from the solar illumination correction.

```

Comment:
"Unphysical" might be confusing to our users. Can we explain this in more detail?

Reply:
Hi @robbibt , I have updated this section after consulting with Fuqin and Aman.